commands.go

```diff
@@ -11,23 +11,25 @@ import (
 )
 
 func registerCommands() {
-	commands = append(commands, commandInfo{"clear", "Clears your terminal", clearCommand, false, false, nil})
-	commands = append(commands, commandInfo{"message", "Sends a private message to someone", messageCommand, false, false, []string{"msg", "="}})
-	commands = append(commands, commandInfo{"users", "Gets a list of the active users", usersCommand, true, false, nil})
-	commands = append(commands, commandInfo{"all", "Gets a list of all users who has ever connected", allCommand, true, false, nil})
-	commands = append(commands, commandInfo{"exit", "Kicks you out of the chat incase your client was bugged", exitCommand, false, false, nil})
-	commands = append(commands, commandInfo{"bell", "Toggles notifications when you get pinged", bellCommand, true, false, nil})
-	commands = append(commands, commandInfo{"room", "Changes which room you are currently in", roomCommand, false, false, nil})
-	commands = append(commands, commandInfo{"kick", "Kicks a user", kickCommand, true, true, nil})
-	commands = append(commands, commandInfo{"ban", "Bans a user", banCommand, true, true, nil})
-	commands = append(commands, commandInfo{"id", "Gets the hashed IP of the user", idCommand, true, false, nil})
-	commands = append(commands, commandInfo{"commands", "Get a list of commands", commandsCommand, false, false, []string{"commands"}})
-	commands = append(commands, commandInfo{"nick", "Change your display name", nickCommand, false, false, nil})
-	commands = append(commands, commandInfo{"color", "Change your display name color", colorCommand, false, false, nil})
-	commands = append(commands, commandInfo{"timezone", "Change how you view time", timezoneCommand, false, false, []string{"tz"}})
-	commands = append(commands, commandInfo{"emojis", "Get a list of emojis you can use", emojisCommand, false, false, nil})
-	commands = append(commands, commandInfo{"unban", "Unban a user", unbanCommand, false, true, nil})
-	commands = append(commands, commandInfo{"help", "Get generic info about the server", helpCommand, false, true, nil})
+	clear := commandInfo{"clear", "Clears your terminal", clearCommand, false, false, nil}
```

Comment:
I highly suggest writing a register() function that appends commands, instead of making so many variables

If you _still_ feel like using so many variables, at least wrap them in a `var (...)` so gofmt can align them.